src/pages/operations/fleet.mdx

```diff
@@ -7,32 +7,12 @@ import TurkishAirlinesCards from '../../components/Operations/Fleet/TurkishAirli
 <div class="hero shadow--lw">
   <div class="container">
     <h1 class="hero__title">Fleet</h1>
-    <p class="hero__subtitle">Flying or somethin</p>
+    <p class="hero__subtitle"></p>
   </div>
+  The Turkish Virtual fleet consists of a range of aircraft that includes the Airbus A321-200, Boeing B737-900, Airbus A330-300, Boeing B777-300, Boeing 787-9, and the Airbus A350-900.
```

Comment:
Text should be completely outside of the div tag or any HTML tags. The body -- the markdown -- is where the Lorem Ipsum filler text was.